.devcontainer/docker-compose.yml

```diff
@@ -41,3 +41,7 @@ services:
 
     # Add "forwardPorts": ["5432"] to **devcontainer.json** to forward PostgreSQL locally.
     # (Adding the "ports" property to this file will not forward from a Codespace.)
+
+networks:
+  print-network:
+    external: true
```

Comment:
as in the end you do not use the external print network because you left `network_mode`, you should remove this.